Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Enter person " + (i+1) + " first name:");
+                string fName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + "'s last name:");
+                string lName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s age:");
+                int age = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s weight in kg:");
+                double weight = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s height in cm:");
+                double height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine(fName + " " + lName + "is " + age + " years old, their weight is " + weight + " kg and their height is " + height + " cm.");
+
+                float bmi = weight / Math.Pow(height / 100.0, 2);
```

Comment:
This causes an error now. You probably need to use (height * height * 10000) instead so the data type is float and not double.

Reply:
Would it look like:

float bmi = weight / (height * height * 10000);

Reply:
Yes! :)